src/test/java/org/folio/rest/api/OaiPmhViewTest.java

```diff
@@ -385,6 +402,74 @@ public void testResponseStatus400WhenRequestingWithInvalidUntilDate()
     });
   }
 
+  @Test
+  public void test500(TestContext testContext) {
```

Comment:
These tests don't appear to be related to the OAI-PMH view and instead are integration tests for the abstract base class used to allow for streaming fetches. I think these tests could be moved to a separate class to reflect this.